suite-native/atoms/src/Hint.tsx

```diff
@@ -15,6 +15,9 @@ type HintProps = {
     children?: ReactNode;
 };
 
+const ICON_SIZE = 14;
+const SPACE_SIZE = 6;
```

Comment:
is there a reason to have these numbers in form of variables, when they are used only one place?

Reply:
I prefer to use descriptive consts for these numbers, so it is easier to review and reuse them. In this case, both of them are used only once and it is easy to read what the numbers mean, so it is not necessary. But I would prefer to keep it this way if you don't mind.

Reply:
I understand the approach, but reading what number was used is more difficult with const than direct use. 

And in case it is used on one place only it (IMO) feels harder to understand the code. But if you prefer it this way, I dont have a strong opinion on this. Feel free to resolve

Reply:
Just to clarify my point of view... I see the numbers inside of the JSX same as an inline style. IMO all these stylings and configurations should be at the beginning of the file.